lib/AST/Attr.cpp

```diff
@@ -131,6 +131,48 @@ DeclAttributes::isUnavailableInSwiftVersion(
   return false;
 }
 
+const AvailableAttr *DeclAttributes::getPotentiallyUnavailable(
+                          const ASTContext &ctx) const {
```

Comment:
Nitpick: LLVM style says to just double-indent this, rather than trying to line it up with something. (Or whatever clang-format does on our repo.)

Reply:
This particular decl is probably best wrapped after the return type, but thank you—I've been struggling to figure out what to do when a function name gets this long.